### ISSUE TITLE
rc-collapse not accessible

### ISSUE THREAD

**Issue by mellis481** [fis-components/rc-collapse#1]
@2betop `rc-collapse`'s collapse/expand functionality only works on the ENTER key keyboard press.  To be accessible, it must work with SPACEBAR or else it breaks [WCAG 2.4.7 Focus Visible](https://www.w3.org/WAI/WCAG21/quickref/#focus-visible) and possibly [WCAG 2.1.1 keyboard](https://www.w3.org/WAI/WCAG21/quickref/#keyboard).

You can see in the WCAG examples, the collapsible elements work with SPACEBAR.  Can `rc-collapse` be updated to work with SPACEBAR?

**Comment by mellis481** [comment]
@ascoders Can you respond to this issue please?
